_posts/2024-01-16-owasp-top10-with-abstract-examples-part-2.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: OWASP top ten 2021 explained with non-technical examples, </br>Part 2
+author: petteri.poyhtari
+excerpt: >
+   How could we better make the most typical risks of web applications aware of all project personnel, in order to improve the quality of the entire web infrastructure. Part 2 blog post about OWASP top 10 with physical world analogies.
```

Comment:
This excerpt is difficult to parse... a more direct and concise wording could improve readability.
"in order to improve" => "to improve" 

This asks a "how" question without a "?", it could be better as a statement like "Learn the most typical web application security risks to improve the quality."

Reply:
Hyvä nosto monimutkaisuudesta. Jätin kuitenkin tuon "miten saataisiin non tec ihimisille" kyssärin, mutta vähän yksinkertaistin lausetta.